documentation/src/main/markdown/downloadingandrunning/ephemeralscan.md

```diff
@@ -6,7 +6,10 @@ Enable this feature by adding [--detect.blackduck.scan.mode=EPHEMERAL](../proper
 
 ## Requirements and Limitations
 
- * A limited subset of Tools can be run. 
+ * Ephemeral scanning is available under “Limited Customer Availability (LCA)”.
+ * Must be running [blackduck_product_name] 2022.10.0 or greater using the hosted KB.
+ * Have Match as a Service (Maas) enabled which will be available with the [blackduck_product_name] 2022.10.0 release.
```

Comment:
Even though it's clear to us, I'd suggest clarifying that "Have Match as a Service (Maas) enabled" is *in Black Duck*. I can imagine users wondering how to enable it in Detect.